packages/react/src/patterns/blocks/ContentBlockSimple/README.md

```diff
@@ -0,0 +1,84 @@
+# Content Block - Simple
+
+> The Content Block - Simple pattern is a decorator of `ContentBlock`, and
+> includes a single `ContentItem`, optional media (image), and ends with a CTA.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/content-block-simple';
+```
+
+> 💡 Only import fonts once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { SimpleLongForm } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  return (
+    <ContentBlockSimple
+      heading={heading}
+      copy={copy}
+      items={items}
```

Comment:
I think we can get rid of this now.

Reply:
Yes this has already been updated.

Reply:
The README here hasn't been updated though.